Migle - MiglePetrou/week6_readme.md

```diff
@@ -0,0 +1,9 @@
+#mp_week6
```

Comment:
Hey! I am having trouble commenting on your Google Sheet directly, so below are all my comments. The numbers are for ease of reference (and do not refer to survey question numbers). Please let me know if you have any questions/clarifications/comments. Overall, nice job!

1. What is your target demographic? All adults? Americans? Non-Americans? Etc.

2. Intro page has an errant carriage return. Nice inclusion of estimated time for survey. Also, it would be nice to combine the intro page with the consent question for brevity.

3. When I press "no" on the consent page, it still asks me to do the survey. This seems like a problem.

4. For the purposes of data standardization, it would be prudent to specify the format of the "what is your name?" question. E.g., "first and last," "last, first," or something of that nature.

5. Nice name and age confirmation.

6. On the race question, it would be great to include a write-in option if "other" is selected. Maybe not in the scope of the project, though.

7. What is the purpose of the marital status question? You might consider re-framing the question or adding options like "single head of household" to refine data collection.

8. You might consider revising urban/rural question. I see two potential issues: many people won't know their town populations. Many small towns are not rural but, in fact, are suburbs. One way to improve this question is to just for the town/city/ZIP/state/etc. that the respondent lives in, and then make rural/urban classifications using administrative data. This method, I believe, would be more accurate and less prone to errors. Your thoughts?

9. The household-size question explicitly refers to family members, which excludes roommates. Is there a reason you did this? Might want to revise you US Census definition (something like "people who have sleep in your household for more than half of the nights in the past year").

10. In the employment status question, you include "unemployed (not currently looking for opportunities)." Usually (in the US, at least), unemployed = in labor force and looking for work. If someone is neither currently working nor looking for work, that person is called "out of the labor force."

11. Also, on the employment status question, consider defining full-time vs. part-time work. For example, US Census considers 35+ hours full-time employment, although some may think it should be 40+ hours, and other nations use different standards.

12. I notice you ask for current employment status but you ask for income *last* calendar year. Is there a reason you made this choice, instead of making both questions current or last year?

13. I get the difference you are drawing between personal vs. household income, but it's not clear until you get to the household income question. Consider adding that clarification on the first question. Also, why did you use different intervals for household income?

14. Regarding the education question, I do not know what "lower secondary school" means. Also, a vocational degree is typically completed *after* high school, so the ordering of the answers is not intuitive, if your target demographic includes Americans. Not sure what nationality you're targeting, but one way to avoid nuances in schooling categories would be to simply ask how many years of education the respondent has had. Also, this is a required question—might want to make option.

15. On your political leanings question, I would recommend some changes. First, adding Libertarian and Green as options. Second, you might alternatively add / change the question to a spectrum like "very conservative, conservative, somewhat conservative, moderate, somewhat liberal, liberal, very liberal." It would also be cool to make this question into two: one about leanings on economic policy, the other about social policy.

16. Strongly urge you to provide "prefer not to say" option for party affiliation, voting history, and voting intention questions. Very sensitive topics in the US. Pollsters consistently find rightward-leaning Americans are less likely to respond truthfully or at all to political polling questions, and requiring these questions would likely create systematic sampling bias.

17. Economic spectrum question is great (I suggested it above before I got to it). However, it seems out of order. Perhaps move it up near the party-affiliation question.

18. Question on monthly heating spending is confusingly phrased—what do you mean by "accommodation"? Also, heating prices obviously vary a TON by season and geography. I would highly recommend including questions on such matters. Lastly, consider adding "not sure / prefer not to say" option. Many renters, for example, have heat costs built into their rent, and people who use electricity to heat home often can't distinguish between heating costs and other uses of electricity.

19. Is there a reason you specified round trip flights? Also, you will likely under-count flights because of the particular COVID-related travel restrictions in 2020 and 2021.

20. Eating habits are sensitive. Recommend making option / including "prefer not to say" option. Also, the answer "almost or at least daily" is confusing. Almost daily = most but not all days. At least daily = one or more times per day.

21. The question on transportation "to work/university" doesn't include an N/A option, even though the survey universe includes unemployed and out of labor force. And the question is required. These facts create guaranteed false responses and/or systematic attrition bias. In general, highly recommend you make everything but the most essential questions optional.

22. Especially since the pandemic, many Americans get groceries delivered, so the grocery-store-transportation question doesn't have answers that accurately describe them. The result would be false answers in the survey, since the question is required.

23. The public-transit-accessibility question doesn't apply to people who live in places without public transit (i.e., anywhere outside a city). This question, because it is required, would therefore generate false responses or systematic attrition bias.

24. The slide that says "think about some the habits you think are most harmful to environment." Is this meant to include a question? Also, the way this is phrased is likely to be off-putting/accusatory to many respondents.

25. Question asking "how many of the following you engage in" is missing a question mark. Also, what defines a long shower? Are you asking if the respondent has ever driven a car or does so regularly? Also, don't you already know this information from previous questions?

26. On the repeating loop questions about habits, it seems inefficient to have the respondent write out each one. Why not just have the respondent select each one from a list? Also, the survey allows the respondent to write any habit, even one not listed.

27. The "how often you engage in the habit" question is unclear and not conducive to standardization. Are you looking for a number of hours?

28. For the second, third, and fourth habits, the survey says "please name the first habit (habit #)." This is an error.

29. The "is climate change real?" question is very loaded and will create a lot of attrition bias in the survey. What do you mean by real? What would you consider someone who believes climate change is happening but primarily due to natural causes (you consider this later but not here)? What about people who are unsure (there are a lot)? Given this question is at the core and crux of your survey description, I highly recommend considering more precise language and more clarifying questions.

30. On the "how often do you discuss / think about climate change" question, why are your time intervals different from those you've previously used? Might consider standardizing them, which would be conducive to data analysis later on.

31. Your scale of "not at all, a little, moderately, a lot, greatly" is likely to confuse respondents. E.g., is "a lot" more than "greatly"? Recommend checking out some of the polls published by the Pew Research Center for sample formats. Also add "unsure" option.

32. Define "ambitious policy actions." Rings of very technocratic language that is vague to people outside the Beltway.

33. Define "limit flying," "limit driving," "limit beef consumption," etc. Vague. How large reduction? I doubt most respondents could assess feasibility without more details.

34. You said the respondent will be asked about two things but then you listed at least five (flying, driving, and switching to fuel-efficient/electric, beef, heating/cooling).

35. Fuel-efficient/electric question does not apply to those without cars. Because this is a required question, there will necessarily be false responses.

36. The question about how important "ambitious climate policies" are to adopting a "sustainable lifestyle" is very vague. Also, I'd include the full question on each page, as respondents may forget what you're asking (at least I did).

37. I have the same constructive feedback as above for most of the questions in this module. Also, who are "the wealthiest"? And what "behavior" change are you referring to?

38. "Investments would concern renewable power plants, ..." would they concern or would the investments be in those things? Unclear. Also, would these investment be in public or private infrastructure? For example, would the public debt go toward "thermal renovation" of for-profit corporate offices? Unclear, and very relevant to the later fairness question.

39. The language of winners/losers is pretty specific to policy/economic academic circles. Consider be more explicit in your language. E.g., do you think low-income earners would financially benefit from XYZ?

40. Your question about other sources of funding is very confusing. Public debt is usually paid eventually, which means increasing taxes, whether it be sales tax, taxing wealthy, etc. So separating out those options is strange/confusing, without further clarity. Also, did you consider adding a private funding option? Or what about making infrastructure green through corporate regulation rather than funding?

41. Question about additional funding requires respondent select at least one answer. This will produce false answers for those who oppose the program.